lexicons/tools/ozone/moderation/defs.json

```diff
@@ -395,7 +395,7 @@
     "accountEvent": {
       "type": "object",
       "description": "Logs account status related events on a repo subject. Normally captured by automod from the firehose and emitted to ozone for historical tracking.",
-      "required": ["timestamp", "status"],
+      "required": ["timestamp", "status", "active"],
```

Comment:
Adding a required field means that old events missing this field will become invalid.  Is that fine in this case?

Reply:
Oh haha nevermind!  I was reading diff from recent commits and forgot this event was brand new in the PR 🙃